.cirrus.yml

```diff
@@ -70,8 +70,8 @@ task:
 
 task:
   name: Test Zstd MacOS
-  osx_instance:
-    image: monterey-xcode
+  macos_instance:
+    image: ghcr.io/cirruslabs/macos-ventura-xcode:14.1
```

Comment:
Since this fails we can probably keep it as it was. We anyway plan to remove the old build system soon

Reply:
The previous task also failed since 1st of January. Maybe we should remove it instead no ?